floor/test/integration/boolean_conversions/bool_test.dart

```diff
@@ -64,6 +64,23 @@ void main() {
       final actual = await boolDao.findWithNullable(false);
       expect(actual, equals(obj));
     });
+
+    test('return nullables', () async {
+      await boolDao
+          .insertBoolC(BooleanClass(false, nullable: null, nonnullable: true));
+      await boolDao
+          .insertBoolC(BooleanClass(true, nullable: false, nonnullable: true));
+
+      expect(await boolDao.getNullables(), equals([null, false]));
+    });
+    test('return nullables', () async {
```

Comment:
💡  The test's name doesn't reflect the actual test.

```suggestion

    test('return nonNullables', () async {
```

Reply:
will fix!